### PR TITLE
Implement segment io

### PR DESCRIPTION
**Pull request by alejandroweb**
### What

We'll start using Segment.io instead of Mixpanel in our frontend apps. Mixpanel will continue working but through Segment.

Segment allows tracking of all your data once, you can send it to any tool your want to use.

https://segment.com/

### How

- Removing Mixpanel library and adding `analytics.js` that it's from segment
- Removing all the mixpanel logic that is not needed

### CODE REVIEW

File: test/test.html [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+    !function () {
+      var analytics = window.analytics = window.analytics || []; if (!analytics.initialize) if (analytics.invoked) window.console && console.error && console.error("Segment snippet included twice."); else {
+        analytics.invoked = !0; analytics.methods = ["trackSubmit", "trackClick", "trackLink", "trackForm", "pageview", "identify", "reset", "group", "track", "ready", "alias", "debug", "page", "once", "off", "on"]; analytics.factory = function (t) { return function () { var e = Array.prototype.slice.call(arguments); e.unshift(t); analytics.push(e); return analytics } }; for (var t = 0; t < analytics.methods.length; t++) { var e = analytics.methods[t]; analytics[e] = analytics.factory(e) } analytics.load = function (t) { var e = document.createElement("script"); e.type = "text/javascript"; e.async = !0; e.src = ("https:" === document.location.protocol ? "https://" : "http://") + "cdn.segment.com/analytics.js/v1/" + t + "/analytics.min.js"; var n = document.getElementsByTagName("script")[0]; n.parentNode.insertBefore(e, n) }; analytics.SNIPPET_VERSION = "4.0.0";
+        analytics.load("AnyxQc63PqJaoMsqlxSRFraqrfsuZr2i");
+        analytics.page();
```
Comment (lapenya) [inline test/test.html]: do we need this `page()`?

File: src/js/tracking-wallet.js [hunk 2]
```diff
@@ -563,7 +539,7 @@
         _lastTouchUTMTags();
       }
 
-      _peopleSetParams();
+      window.analytics.identify();
```
Comment (lapenya) [inline src/js/tracking-wallet.js]: empty `identify()` without traits?

File: test/test.html (hunk 1 above)
Comment (alejandroweb) [inline test/test.html]: yep it's the demo page so no problem to have it empty. I think is enough for testing purposes.  We are testing form submission here.

File: src/js/tracking-wallet.js (hunk 2 above)
Comment (alejandroweb) [inline src/js/tracking-wallet.js]: My first approach was initialize the library with anonymous users and later do an identify. It's true that if we have the user info, start with the user info. Better approach so changed ;)

File: test/test.html (hunk 1 above)
Comment (alejandroweb) [inline test/test.html]: and every page must have a `.page()` event as the docs says

**Pull request merged**